docs/installation/docker.md

```diff
@@ -69,8 +69,7 @@ you want to update that set the `POSTGRES_VERSION` variable in your environment
 and start the container again_
 
 1. run `docker-compose build`
-1. run `docker-compose run web rails db:setup`
-1. run `docker-compose run web rails search:setup`
+1. run `docker-compose run web rails db:setup db:migrate search:setup`
```

Comment:
Reasons for this change:

1. Docker is slow on macOS, by chaining the various tasks we only need to load the Rails environment once instead of several times.
2. If someone is trying Docker for the first time on a branch that has unmerged migrations, they need to run `db:migrate`, so it's just easier to include it.

Reply:
Great idea! We might want to eventually have a `docker.rake` task that invokes all of those in the right order.

Reply:
Great idea @rhymes!